appveyor.yml

```diff
@@ -1,5 +1,5 @@
 version: '{branch}.{build}'
-shallow_clone: true
+clone_depth: 3
```

Comment:
Previously, Appveyor would download the repo at a specific commit as a zip file. By setting the `clone_depth` (to the same level as Travis), we actually clone the repo. This is necessary because we need to do `git diff` to figure out which files have changed.